app/src/main/java/com/keylesspalace/tusky/components/filters/FiltersActivity.kt

```diff
@@ -81,9 +84,12 @@ class FiltersActivity : BaseActivity(), FiltersListener {
                             binding.messageView.show()
                         } else {
                             binding.messageView.hide()
-                            binding.filtersList.adapter = FiltersAdapter(this@FiltersActivity, state.filters)
                             binding.filtersList.show()
                         }
+
+                        if (binding.filtersList.adapter == null || (binding.filtersList.adapter as FiltersAdapter).filters != state.filters) {
```

Comment:
Why not just `?.`?

Reply:
How exactly would that look?

Reply:
Sorry, missed this reply in my notifications.
I'm thinking like `(binding.filtersList.adapter as FiltersAdapter)?.filters != state.filters` - but it's just a minor nitpick